src/sugar3/graphics/radiopalette.py

```diff
@@ -36,24 +40,38 @@ def __init__(self, **kwargs):
         self.connect('notify::palette', self.__palette_cb)
 
     def __palette_cb(self, widget, pspec):
+        '''
+        A callback function
+        '''
         if not isinstance(self.props.palette, RadioPalette):
             return
         self.props.palette.update_button()
 
 
 class RadioToolsButton(RadioMenuButton):
+    '''
+    This is a button used to open a group of widgets in a RadioPalette(menu)
+    It is clickable
+    '''
 
     def __init__(self, **kwargs):
         RadioMenuButton.__init__(self, **kwargs)
 
     def do_clicked(self):
+        '''
+        This function is called when RadioToolsButton is clicked
+        '''
         if not self.selected_button:
             return
         self.selected_button.emit('clicked')
 
 
 class RadioPalette(Palette):
-
+    '''
+    This contains the widgets to be displayed in the RadioMenuButton
+    and RadioToolsButton. This is the Palette (menu) that shows up when
+    those buttons are clicked. It is used to display the widgets.
```

Comment:
Great!  But how does this differ from a normal palette?

Also, a usage example (just copy and paste one from your big example script) would help tons.